notebooks/214-grammar-correction/214-grammar-correction.ipynb

```diff
@@ -62,12 +62,12 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import ipywidgets as widgets
</pre><p>I think as user has opportunity to modify text in gradio demo, there is no need in widget, just use plain text instead, otherwise we have 2 different ways to launch model with user input</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Will it affect the project documentation?

Reply:
what do you mean under documentation?
Probably it should not, it just removing dependency of notebook from pywidgets.

In part before running radio you just make sure that pipeline work, so constant input should be ok for these purposes as you have gradio for providing input. If under documentation, you mean description inside notebook, probably it requires small changes with removing sentence which says that user should provide own text in widget window (I do not remember exactly)

Reply:
Done